doc/apiary/v2/fiware-ngsiv2-cookbook.apib

```diff
@@ -397,16 +397,18 @@ This resource allows you to obtain a list of subresources published by the API:
 
 ## Query Entities [/v2/entities{?q,limit,offset,coords,geometry,attrs,options}]
 
-### Value filter and Geoquery [GET /v2/entities{?q,limit,offset,coords,geometry,attrs,options}]
+### Value filter and Geoquery [GET /v2/entities{?q,limit,offset,coords,georel,geometry,attrs,options}]
 
 + Parameters
     + q: temperature==22.5;humidity==60 (optional, string) - General query parameter that allows
     to filter by attribute content
-    + coords: 41.3763726, 2.1864475 (optional, string) - Coordinates. Comma separated values.
-    The number of values will depend on the 'geometry' attribute.
-    + geometry: circle;radius=10000 (optional, string) - Geospatial area for the query. 
-    The 'exterior' modifier allows to indicate that the referred area is the outside
-    of the region, for instance, 'polygon;exterior'
+    + coords: 41.3763726, 2.1864475 (optional, string) - Coordinates of a reference geometry used for the query.
+    Comma separated values. The number of values and its interpreation depends on the `geometry` attribute.
+    + geometry: point (optional, string) - Geometry (point, line, polygon) represented by the `coords` parameter. 
+    + georel: near;maxDistance:1000 (optional, string) - Geospatial relationship that must be met between
+    the reference geometry (specified by the `coords` and `geometry` parameters) and the matching entities.
+    This example refers to those entities located at a distance of 1000 meters at a maximum
+    (from the point specified as reference geometry).  
     + limit: 20 - Limit the number of results (optional, number) - Maximum number of results to return
     + attrs: seatsNumber (optional, string) - Attributes to be returned
     + options: count,keyValues (optional, string) - query options (count total, keyValues mode)
```

Comment:
you are right. dropped at https://github.com/telefonicaid/fiware-orion/pull/1959/commits/f3fd76c3817fd1a5b8f5d1059e37ebd4be19f371 

and considering adding a new example for count in an upcoming PR